workspaces/announcements/.changeset/strong-flowers-remain.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage-community/plugin-announcements-backend': patch
+'@backstage-community/plugin-announcements-common': patch
+'@backstage-community/plugin-announcements-react': patch
+'@backstage-community/plugin-announcements': patch
+---
+
+Slight improvements to README documentation.
+
+Internal only - Adds a minimal backstage instance with the announcements plugins installed. It is intended to be used for testing and development purposes, along with demonstrating all the features of the announcements plugin.
```

Comment:
This part can be dropped, changesets get rolled up into the changelog, having this there won't really make sense. 👍